### DISCUSSION TITLE
Should amiibo drop more than one chest?

### DISCUSSION THREAD

**Discussion by jordanbtucker** [jordanbtucker/totk-no-loot-boxes#9] [Polls]
Since many of you are probably using this mod just to get the rare amiibo items, do you think this mod should drop more than one chest to give you a better chance of getting all rare items for that amiibo in one use?

For example, the Skyward Sword Link amiibo can drop up to five different rare items with this mod, each with a 20% probability. If this mod dropped 15 chests at once, then there's about an 83% chance you'll get all items in just one amiibo use. This means you'd only need to try the amiibo a max of about three times with save scumming to get all rare items from it.

**Comment by jordanbtucker** [comment]
Votes are coming in and it's obvious so far that more chests is the preferred option, but we're split on whether 15 chests is too many. Leave a comment on how many chests you think should drop. 10, 8, 5, 3?

**Reply by newakuma** [reply]
5 is enough

**Comment by gatlinnewhouse** [comment]
5

**Comment by Stormsinger-Frostclaw** [comment]
I'm all for 5 chests.

**Comment by Kelbitron** [comment]
I agree, 5 seems right

**Comment by jordanbtucker** [comment]
It's still split between 15 chests being right and 15 chests being too much. Maybe there's a middle ground where I include an infinite amiibo cheat with the mod for those that want it?
